src/contracts/core/RewardsCoordinator.sol

```diff
@@ -303,23 +303,35 @@ contract RewardsCoordinator is
         require(msg.sender == operator, "RewardsCoordinator.setOperatorAVSSplit: caller is not the operator");
         require(split <= ONE_HUNDRED_IN_BIPS, "RewardsCoordinator.setOperatorAVSSplit: split must be <= 10000 bips");
 
-        uint32 activatedAt = uint32(block.timestamp) + activationDelay;
-        uint16 oldSplit = _getOperatorSplit(operatorAVSSplitBips[operator][avs]);
-        _setOperatorSplit(operatorAVSSplitBips[operator][avs], split, activatedAt);
+        (uint16 oldSplit, uint32 oldActivatedAt) = _getOperatorSplit(operatorAVSSplitBips[operator][avs]);
+        require(
+            block.timestamp > oldActivatedAt,
+            "RewardsCoordinator.setOperatorAVSSplit: earlier split not activated yet"
+        );
+
+        uint32 newActivatedAt = uint32(block.timestamp) + activationDelay;
 
-        emit OperatorAVSSplitBipsSet(msg.sender, operator, avs, activatedAt, oldSplit, split);
+        _setOperatorSplit(operatorAVSSplitBips[operator][avs], split, newActivatedAt);
+
+        emit OperatorAVSSplitBipsSet(msg.sender, operator, avs, newActivatedAt, oldSplit, split);
     }
 
     /// @inheritdoc IRewardsCoordinator
     function setOperatorPISplit(address operator, uint16 split) external onlyWhenNotPaused(PAUSED_OPERATOR_PI_SPLIT) {
         require(msg.sender == operator, "RewardsCoordinator.setOperatorPISplit: caller is not the operator");
         require(split <= ONE_HUNDRED_IN_BIPS, "RewardsCoordinator.setOperatorPISplit: split must be <= 10000 bips");
 
-        uint32 activatedAt = uint32(block.timestamp) + activationDelay;
-        uint16 oldSplit = _getOperatorSplit(operatorPISplitBips[operator]);
-        _setOperatorSplit(operatorPISplitBips[operator], split, activatedAt);
+        (uint16 oldSplit, uint32 oldActivatedAt) = _getOperatorSplit(operatorPISplitBips[operator]);
+        require(
+            block.timestamp > oldActivatedAt,
+            "RewardsCoordinator.setOperatorPISplit: earlier split not activated yet"
+        );
+
+        uint32 newActivatedAt = uint32(block.timestamp) + activationDelay;
```

Comment:
nit: why rename from `activatedAt` to `newActivatedAt`?

Reply:
Fixed in [b2f8bdd](https://github.com/Layr-Labs/eigenlayer-contracts/pull/947/commits/b2f8bdd1c452c037f9db385fe529fb08da2d8ad8)